### PR TITLE
dx-1066 Screenshots for payments pages. Also some minor language fixes.

### PR DESCRIPTION
**Pull request by arebra**
Will add checkout tomorrow morning

### CODE REVIEW

**Review by asbjornu — APPROVED** [review]
Not only doing what's written on the tin, but fixing typos and such elsewhere as well. Stellar work!

**Review by marinafrydenb — APPROVED** [review]
nice cleanup!

File: checkout/payment-menu.md [hunk 1]
```diff
@@ -297,8 +305,10 @@ finalizing the payment in the [After Payment section][after-payment].
                          next_title="Next: Capture" %}
 
 [after-payment]: after-payment
-[guest-payment-menu-image]: /assets/img/checkout/guest-payment-menu-450x850.png
-[login-payment-menu-image]: /assets/img/checkout/logged-in-payment-menu-450x900.png
+[guest-payment-menu-image]: /assets/img/checkout/guest-payment-menu.png
+[login-payment-menu-image]: /assets/img/checkout/loggedin-payment-menu.png
+[swedish-guest-payment-menu-image]: /assets/img/checkout/swedish-guest-payment-menu.png
+[swedish-login-payment-menu-image]: /assets/img/checkout/swedish-logged-in-payment-menu.png
```
Comment (payex-dev) [inline checkout/payment-menu.md:311]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6057883)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6057883&bid=19843700)

File: checkout/payment-menu.md [hunk 2]
```diff
@@ -297,8 +305,10 @@ finalizing the payment in the [After Payment section][after-payment].
                          next_title="Next: Capture" %}
 
 [after-payment]: after-payment
-[guest-payment-menu-image]: /assets/img/checkout/guest-payment-menu-450x850.png
-[login-payment-menu-image]: /assets/img/checkout/logged-in-payment-menu-450x900.png
+[guest-payment-menu-image]: /assets/img/checkout/guest-payment-menu.png
+[login-payment-menu-image]: /assets/img/checkout/loggedin-payment-menu.png
+[swedish-guest-payment-menu-image]: /assets/img/checkout/swedish-guest-payment-menu.png
```
Comment (payex-dev) [inline checkout/payment-menu.md:310]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6057883)

File: _includes/payment-order-other-features.md [hunk 3]
```diff
@@ -231,6 +231,74 @@ Content-Type: application/json
 }
 ```
 
+{% if include.documentation_section == "payment-menu" %}
+
+## Payer Aware Payment Menu
```
Comment (asbjornu) [inline _includes/payment-order-other-features.md:236]: Should `Payer Aware Payment Menu` be added in this PR? I thought @marinafrydenb added that in #598?

**Review by asbjornu — CHANGES_REQUESTED** [review]
Some unrelated changes seems to have snuck in here somehow, otherwise this looks good!

File: _includes/payment-order-other-features.md (hunk 3 above)
Comment (arebra) [reply _includes/payment-order-other-features.md:236]: No it should be a part of this PR. Most likely a merging mishap or something like that.

**Review by asbjornu — APPROVED** [review]

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6057883&bid=19843700)

File: checkout/payment-menu.md (hunk 2 above)
Comment (payex-dev) [inline checkout/payment-menu.md:310]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6057883)

File: checkout/payment-menu.md (hunk 1 above)
Comment (payex-dev) [inline checkout/payment-menu.md:311]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6057883)

**Review by marinafrydenb — APPROVED** [review]

**Pull request merged**